3.11/appendix-telemetrics.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: default
+description: >-
+  ArangoDB automatically gathers information on how it is used and the features
+  being utilized, but you can disable this data collection
+---
+# Telemetrics
+
+{{ page.description }}
+{:class="lead"}
+
+ArangoDB gathers metrics by default, to identify the primary usage patterns and
+features, and to measure their adoption rate.
+
+It is important to note that the information collected by ArangoDB is
+non-personal and purely statistical and helps ArangoDB determine which
+features are most used, how the general usage model looks like, and
+whether ArangoDB is configured properly.
+
+This data is anonymous and does not contain any personal information like
+usernames or IP addresses that could be used to identify a particular user, nor
+does it contain any content of the documents stored in ArangoDB.
+This means that your privacy is protected, and that there is no risk of your
+data being compromised.
+
+The data is transmitted to ArangoDB Inc. or one of its subsidiaries.
```

Comment:
I added this because I believe we need to cover the aspect of sending the data to us, and who "us" is. I would also like to add something like "the information is never shared with other companies".